### PR TITLE
pods: read from task log not executor

### PR DESCRIPTION
**Pull request by tamarrow**
https://mesosphere.atlassian.net/browse/DCOS-10828

### CODE REVIEW

**Comment by kensipe** [comment]
@tamarrow a couple of doc issues but otherwise looks really good!

**Comment by tamarrow** [comment]
thanks @kensipe! fixed the docs issues

File: cli/tests/integrations/common.py [hunk 1]
```diff
@@ -522,15 +538,78 @@ def app(path, app_id, wait=True):
...
+    """Context manager that deploys pods on entrance, and removes
+    them on exit.
+
+    :param pods: dict of path/to/pod/json -> pod id
+    :type pods: {}
+    :rtype: None
+    """
+
+    for pod_path in pods:
+        add_pod(pod_path)
```
Comment (cruhland) [inline cli/tests/integrations/common.py]: Don't we want `wait=False` here?

File: cli/tests/integrations/common.py [hunk 2]
```diff
@@ -151,11 +151,11 @@ def wait_for_service(service_name, number_of_services=1, max_count=300):
         count += 1
 
 
-def add_job(app_path):
+def add_job(job_path):
```
Comment (kensipe) [inline cli/tests/integrations/common.py]: the issue with the build is app_path was changed to "job_path" on line 154, however line 164 also needs to change to job_path.   passes syntax check after the change.

File: cli/tests/integrations/common.py (hunk 1 above)
Comment (kensipe) [inline cli/tests/integrations/common.py]: @tamarrow I totally agree with charles here as well!

**Pull request merged**